main.js

```diff
@@ -0,0 +1,142 @@
+import axios from "axios";
+
+// 1. Data Fetchen
+// Create reference
+const list = document.getElementById("list-of-users");
+
+// Verwijzing naar error message
+const errorMessage = document.getElementById("error-message");
+
+// Asynchrone functie (normale syntax):
+async function fetchData(name) {
+    let URI = "https://restcountries.com/v2/all";
```

Comment:
Variabelnamen in camelCase schrijven
```suggestion
    let uri = "https://restcountries.com/v2/all";
```